src/Take.Blip.Builder/Storage/FromToIdentityInputExpirationPair.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Lime.Protocol;
+
+namespace Take.Blip.Builder.Storage
+{
+    public class FromToIdentityInputExpirationPair
+    {
+        public Identity FromIdentity { get; set; }
+
+        public Identity ToIdentity { get; set; }
+
+        public override string ToString() => $"inputexpiration-loop-counter:{FromIdentity}:{ToIdentity}";
```

Comment:
rename this method to a name that represent the action... for example: GetKey....
this implementation should be make in Template Hosting since is a Redis Key...